packages/core/rendering/core-rendering-server-internal/src/rendering_service.tsx

```diff
@@ -109,6 +111,11 @@ export class RenderingService {
       defaults: uiSettings.globalClient?.getRegistered() ?? {},
       user: isAnonymousPage ? {} : await uiSettings.globalClient?.getUserProvided(),
     };
+    const userSettings = {
+      defaults: {},
+      user: isAnonymousPage ? {} : await uiSettings.userClient?.getUserProfileSettings(),
```

Comment:
nit: silly optimization: fetch the 3 levels concurrently. 

IMO page load should improve.

Reply:
Added in latest commit - let me know if that is what you meant!